src/main/java/uk/gov/justice/digital/job/DataHubJob.java

```diff
@@ -87,15 +87,15 @@ private void batchProcessor(JavaRDD<byte[]> batch) {
             getTablesInBatch(dataFrame).forEach(table -> {
                 try {
                     val dataFrameForTable = extractDataFrameForSourceTable(dataFrame, table);
+                    dataFrameForTable.persist();
 
-                    val rawDataFrame = rawZone.process(spark, dataFrameForTable, table);
-                    rawDataFrame.persist();
+                    rawZone.process(spark, dataFrameForTable, table);
 
-                    val structuredDataFrame = structuredZone.processLoad(spark, rawDataFrame, table);
-                    curatedZone.processLoad(spark, structuredDataFrame, table);
+                    val structuredLoadDataFrame = structuredZone.processLoad(spark, dataFrameForTable, table);
+                    curatedZone.processLoad(spark, structuredLoadDataFrame, table);
 
-                    val structuredIncrementalDataFrame = structuredZone.processCDC(spark, rawDataFrame, table);
-                    curatedZone.processCDC(spark, structuredIncrementalDataFrame, table);
+                    val structuredCDCDataFrame = structuredZone.processCDC(spark, dataFrameForTable, table);
```

Comment:
can we rename the variable structuredCDCDataFrame? it appears CD twice

Reply:
Thanks. I'll find a suitable name for it and amend in my next PR.